connector/unit/mapper.py

```diff
@@ -142,7 +142,7 @@ def convert(field, conv_type):
     """
     def modifier(self, record, to_attr):
         value = record[field]
-        if not value:
+        if not value and value != 0:
```

Comment:
there's the same issue with `value == ''` so maybe we should totally change this condition